File: src/main/java/com/randomperson22/piggymodtest/npc/pony/PonyNpcRenderer.java
package com.randomperson22.piggymodtest.npc.pony;

import net.minecraft.client.renderer.entity.RenderManager;
import software.bernie.geckolib3.renderers.geo.GeoEntityRenderer;

public class PonyNpcRenderer extends GeoEntityRenderer<EntityPonyNpc> {
    public PonyNpcRenderer(RenderManager renderManager) {
        super(renderManager, new PonyNpcModel());
        this.shadowSize = 0.5f;
        // Initialize your animation processor here if needed
	}
}

File: src/main/kotlin/com/randomperson22/piggymodtest/npc/pony/PonyNpcRenderer.kt
package com.randomperson22.piggymodtest.npc.pony

import net.minecraft.client.renderer.entity.RenderManager
import software.bernie.geckolib3.renderers.geo.GeoEntityRenderer

class PonyNpcRenderer(renderManager: RenderManager?) :
    GeoEntityRenderer<EntityPonyNpc?>(renderManager, PonyNpcModel()) {
    init {
        this.shadowSize = 0.5f
        // Initialize your animation processor here if needed
    }
}